kubernetes/cray-sysmgmt-health/dashboards_json/timing/timing-dashboard.json

```diff
@@ -0,0 +1,2100 @@
+{
+  "annotations": {
+    "list": [
+      {
+        "builtIn": 1,
+        "datasource": {
+          "type": "prometheus",
+          "uid": "PBFA97CFB590B2093"
+        },
+        "enable": true,
+        "hide": true,
+        "iconColor": "rgba(0, 211, 255, 1)",
+        "name": "Annotations & Alerts",
+        "target": {
+          "limit": 100,
+          "matchAny": false,
+          "tags": [],
```

Comment:
Can we add a tag to filter IUF dashboards.

Reply:
`"refresh": "",
  "schemaVersion": 36,
  "style": "dark",
  "tags": [
    "IUF"
  ],`

I have added it here.